hudi-common/src/main/java/org/apache/hudi/common/util/ParquetUtils.java

```diff
@@ -380,4 +382,76 @@ public Boolean apply(String recordKey) {
 
     return val;
   }
+
+  /**
+   * Parse min/max statistics stored in parquet footers for all columns.
+   */
+  public Collection<HoodieColumnStatsMetadata<Comparable>> readColumnStatsFromParquetMetadata(Configuration conf,
+                                                                                              String partitionPath,
```

Comment:
we already has a function readRangeFromParquetMetadata to read min-max and null count info ,  could you reuse that function.    readRangeFromParquetMetadata  solve the problem of reading datetype type statistics concurrently

Reply:
sure @xiarixiaoyao, will take a look at readRangeFromParquetMetadata.

Reply:
yes, this is addressed. Thanks @xiarixiaoyao